source/common/network/listener_impl.h

```diff
@@ -28,15 +28,13 @@ class ListenerImpl : public BaseListenerImpl {
   ListenerCallbacks& cb_;
 
 private:
-  static void listenCallback(evconnlistener*, evutil_socket_t fd, sockaddr* remote_addr,
-                             int remote_addr_len, void* arg);
-  static void errorCallback(evconnlistener* listener, void* context);
+  void onSocketEvent(short flags);
 
   // Returns true if global connection limit has been reached and the accepted socket should be
   // rejected/closed. If the accepted socket is to be admitted, false is returned.
   static bool rejectCxOverGlobalLimit();
 
-  Event::Libevent::ListenerPtr listener_;
```

Comment:
This type I think can be deleted now wherever it is defined.

Reply:
Ack! Will remove!